app/styles/_library/_objects.touts.scss

```diff
@@ -272,13 +272,13 @@
   }
 }
 .c-newsletter-tout__dek {
-  @include typeface(subheader, 1);
+  @include typeface(subheader, 7);
 
-  font-size: var(--font-size-body-1);
```

Comment:
This is another workaround where we can just set the size we want now
`@include typeface(subheader, 5);`

Reply:
⚡